x/vpool/keeper/pool_state_test.go

```diff
@@ -44,8 +43,9 @@ func TestEditPoolConfig(t *testing.T) {
 	pair := asset.Registry.Pair(denoms.BTC, denoms.NUSD)
 	vpoolStart := types.Vpool{
 		Pair:              pair,
-		QuoteAssetReserve: sdk.NewDec(10 * common.Precision),
-		BaseAssetReserve:  sdk.NewDec(5 * common.Precision),
+		QuoteAssetReserve: sdk.NewDec(10 * common.TO_MICRO),
+		BaseAssetReserve:  sdk.NewDec(5 * common.TO_MICRO),
+		SqrtDepth:         common.MustSqrtDec(sdk.NewDec(5 * 10 * common.TO_MICRO * common.TO_MICRO)),
```

Comment:
In this case, I would put in the literal number we expect rather than rely on a utility function to calculate it for us, because the utility function is also used in the class's code path to calculate this value. If something changes in the utility function, this test case would still pass even though the final value isn't what we expect for this vpool entity.

Reply:
I added a simplified constructor for this. I'll add a ticket to convert over most of the `vpooltypes.Vpool` initializations to use `vpooltypes.NewVpool(types.ArgsNewVpool(...))` to accomplish this more directly. 
- https://github.com/NibiruChain/nibiru/issues/1253
I started working on this in the vpool module, but the PR was becoming large. 

Context: A Slack discussion

![image](https://user-images.githubusercontent.com/51418232/229240512-554a8206-7b57-4c87-a517-50d69a5caeb7.png)